libs/design/sidebar/src/sidebar-viewport/sidebar-viewport.component.ts

```diff
@@ -46,22 +46,24 @@ import { DaffSidebarComponent } from '../sidebar/sidebar.component';
 import { DaffSidebarViewportBackdropComponent } from '../sidebar-viewport-backdrop/sidebar-viewport-backdrop.component';
 
 /**
- * The DaffSidebarViewport is the "holder" of sidebars throughout an entire application.
- * It's generally only used once, like
+ * The DaffSidebarViewportComponent is the "holder" of sidebars throughout an entire application.
+ * It's generally only used once, like:
  *
+ * @example Using the sidebar viewport
  * ```html
  * <daff-sidebar-viewport>
  *    <daff-sidebar></daff-sidebar>
- *    <p>Some Content</p>
+ *    <p>Site content</p>
  * </daff-sidebar-viewport>
  * ```
  *
  * Importantly, its possible for there to be multiple sidebars of many modes
- * at the same time. @see {@link DaffSidebarMode }
+ * at the same time. See {@link DaffSidebarMode}.
  *
  * Since this is a functional component, it's possible to have multiple "open" sidebars
  * at the same time. As a result, this component attempts to gracefully handle these situations.
  * However, importantly, there can only be one sidebar of each mode, on each side, at any given time.
+ *
  * If this is violated, this component will throw an exception.
```

Comment:
Note that this content will be part of the example body

Reply:
So all content below @example is rendered in the example body? How do I separate it?

Reply:
Put it above all example tags